### PR TITLE
JBIDE-19516 New Batch Job XML wizard

### PR DESCRIPTION
**Pull request by scabanovich**

### CODE REVIEW

File: batch/plugins/org.jboss.tools.batch.ui/src/org/jboss/tools/batch/ui/internal/wizard/NewJobXMLCreationWizard.java [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+import org.jboss.tools.common.ui.widget.editor.SwtFieldEditorFactory;
+import org.jboss.tools.common.util.FileUtil;
+
+/**
+ * 
+ * @author Viacheslav Kabanovich
+ *
+ */
+public class NewJobXMLCreationWizard extends BasicNewResourceWizard {
+	public static final String WIZARD_ID = "org.jboss.tools.cdi.ui.wizard.NewBeansXMLCreationWizard"; //$NON-NLS-1$
```
Comment (alexeykazakov) [inline batch/plugins/org.jboss.tools.batch.ui/src/org/jboss/tools/batch/ui/internal/wizard/NewJobXMLCreationWizard.java]: cdi->batch

File: batch/plugins/org.jboss.tools.batch.core/templates/job.xml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<job  id="myJob" xmlns="http://xmlns.jcp.org/xml/ns/javaee" version="1.0">
```
Comment (alexeykazakov) [inline batch/plugins/org.jboss.tools.batch.core/templates/job.xml]: 1. Let's add `<?xml version="1.0" encoding="UTF-8"?>` too
2. Remove one extra space between `<job` and `id="myJob"`

**Comment by alexeykazakov** [comment]
What do you think of adding job ID field to the wizard which is validated and show a warning if there is already a job xml with the same ID?

**Comment by scabanovich** [comment]
Done.

**Pull request merged**